source/adapters/cuda/device.cpp

```diff
@@ -612,6 +612,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
     SupportedExtensions += "pi_ext_intel_devicelib_assert ";
     // Return supported for the UR command-buffer experimental feature
     SupportedExtensions += "ur_exp_command_buffer ";
+    SupportedExtensions += "ur_exp_usm_p2p ";
```

Comment:
could do something like
```
SupportedExtensions += std::string(UR_USM_P2P_EXTENSION_STRING_EXP) + " ";
```
just to re-use the macro definition, although that does come at the cost of the extra std::string constructor

Reply:
It does seem to make more sense to use the macro directly, otherwise I don't see much point in having the macro (but I guess there are other reasons). I don't mind although if I did that it would make sense to also change the "ur_exp_command_buffer " case also to use the corresponding macro.
It doesn't really seem important in any case.

Reply:
Let me know if you want me to make this change though. I just want confirmation that it is OK to change the command buffer one too before making such a change.

Reply:
OK I'll change them both in this PR.

Reply:
OK the problem is that I should make the switch the macro usage in all the plugins e.g. https://github.com/oneapi-src/unified-runtime/blob/main/source/adapters/level_zero/device.cpp#L244 is not using the macros, I'm not even sure they have the macros for all strings.
I could make a separate PR to do this prior to this one, however I want to get this PR merged ASAP because it is a prerequisite for https://github.com/oneapi-src/unified-runtime/pull/1234 which we want in the next release.
I suggest:
1. merging this PR as is
2. fixing all the macro strings in this device query for all plugins in a later PR.

Reply:
alright, I'm ok with this course of action